modules/social_features/social_event/modules/social_event_an_enroll/social_event_an_enroll.info.yml

```
@@ -1,4 +1,4 @@
-name: 'Social Event AN Enroll'
+name: 'Social Event Anonymous Enrollments'
```

```suggestion
name: 'Social Event Anonymous Enrolments'
```

See the `Social Event Anonymous Enrolments Export` as example.